src/main/java/frc/robot/subsystems/drivetrain/SwerveDrive.java

```diff
@@ -112,6 +113,71 @@ public void holonomicDrive(double forward, double strafe, double rotation) {
         setStates(kinematics.toSwerveModuleStates(speeds));
     }
 
+    /**
+     * Move the swerve in the specified direction, rotation and velocity.
+     *
+     * @param forward  the velocity on the X-axis. [m/s]
+     * @param strafe   the velocity on the Y-axis. [m/s]
+     * @param rotation the rotational velocity counter-clockwise positive. [rad/s]
+     */
+    public void defaultHolonomicDrive(double forward, double strafe, double rotation) {
+        ChassisSpeeds speeds = fieldOriented ?
+                ChassisSpeeds.fromFieldRelativeSpeeds(forward, strafe, rotation, Robot.getAngle()) :
+                new ChassisSpeeds(forward, strafe, rotation);
+        setStates(kinematics.toSwerveModuleStates(speeds));
+    }
+
+    /**
+     * Move the swerve in the specified direction, rotation and velocity, but the velocities are relative to the error of the modules.
```

Comment:
You don't need 3 functions for holonomicDrive, just create one function that chooses the velocity based on the rotation, vx and vy